src/boxGrid/_boxGridBox.scss

```diff
@@ -23,19 +23,36 @@
     background-color: $color-coat-of-arms-medium-light;
   }
 
+  &--link {
+    text-decoration: inherit;
+    border: inherit;
+    box-sizing: inherit;
+    color: inherit;
+
+    &:visited {
+      color: inherit;
+    }
+  }
```

Comment:
One thing I'd like these boxes to have (which they didn't before to be fair) is some kind of hover effect as is usual for links and buttons of any kind.